channeldb/db.go

```diff
@@ -364,6 +364,29 @@ func Open(dbPath string, modifiers ...OptionModifier) (*DB, error) {
 	return db, err
 }
 
+// Open opens or creates channeldb using the the kv channeldb selected via
+// golang tags (e.g. kvdb_sqlite, kvdb_etcd, kvdb_postgres or default bolt if
+// nothing is explicitly specified).
+// Any necessary schemas migrations due to updates will take place as necessary.
+func Open(dbPath string, modifiers ...OptionModifier) (*DB, error) {
```

Comment:
I think it'd be better to name this `OpenTestDB` and make the renames in the tests too as otherwise this method could be mistaken for a generic open.